pom.xml

```diff
@@ -58,10 +58,11 @@
     <main.basedir>${project.basedir}</main.basedir>
 
     <version.org.awaitility-awaitility>3.0.0</version.org.awaitility-awaitility>
-    <version.io.opentracing>0.31.0</version.io.opentracing>
-    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.2.0</version.io.opentracing.contrib-opentracing-web-servlet-filter>
-    <version.io.opentracing.contrib-opentracing-tracerresolver>0.1.5</version.io.opentracing.contrib-opentracing-tracerresolver>
-    <version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>0.1.0</version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>
+    <version.io.opentracing>0.32.0</version.io.opentracing>
+    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.2.4-SNAPSHOT</version.io.opentracing.contrib-opentracing-web-servlet-filter>
```

Comment:
Snapshot causing ci build problem.

Reply:
yes - see the comment in the PR description.